examples/ref-implementation/backstage/manifests/install.yaml

```diff
@@ -166,7 +166,7 @@ data:
       locations:
         # Examples from a public GitHub repository.
         - type: url
-          target: https://gitea.cnoe.localtest.me:8443/giteaAdmin/idpbuilder-localdev-backstage-entities/src/branch/main/catalog-info.yaml
+          target: https://gitea.cnoe.localtest.me:8443/giteaAdmin/idpbuilder-localdev-backstage-templates-entities/raw/branch/main/catalog-info.yaml
```

Comment:
rebase this is already fix in main
https://github.com/cnoe-io/idpbuilder/commit/ea9c56e6231c05cc33ed6b33a00bec79008a8a16

Reply:
Yes this is the cherry-pick of that exact commit to this branch I needed it to make sure things were working correctly. The merge to main should be noOp here. Happy to remove it if that is more appropriate.